index.html

```diff
@@ -197,6 +196,47 @@ <h3 class="tech-title">Skills</h3>
             </div>
         </section>
     </section>
+<!-- contact form section -->
+<section class="contact" id="contact">
+    <h2 class="contact-message">
+      I'm always interested in hearing about new projects, so if you'd like
+      to chat please get in touch.
+    </h2>
+    <form
+      action="https://formspree.io/f/mnqrnjnk" method="post" class="contact-form"
+      novalidate id="form" ><small id="error-mail"></small><br>
+      
+      <div class="form-element">
+        <input type="text"  name="name" id="name" maxlength="30" placeholder=" Full Name"/>
+      </div>
+      <div class="form-element">
+        <input type="email" name="email" id="email" placeholder=" Email address"/> 
+      </div>
+      <div class="form-element">
+        <textarea name="message" id="message" maxlength="500" placeholder=" Enter text here" ></textarea>
+      </div>
```

Comment:
### Your contact form
![image](https://user-images.githubusercontent.com/93039359/194321610-7648f23b-7c35-414b-865b-181acb3d493e.png)

### Figma Contact Form
![margins-paddings](https://user-images.githubusercontent.com/93039359/194323282-b88941bf-8ee2-46da-89c2-0ab84ab2e6f7.png)

- [ ] Kindly follow the Figma template and add margins and paddings as shown in the Figma image above with red lines to make your design look professional and to attract the recruiter eyes :+1: